src/test/java/it/gov/pagopa/reward/repository/HpanInitiativesAtomicOpsRepositoryImplTestIntegrated.java

```diff
@@ -0,0 +1,11 @@
+package it.gov.pagopa.reward.repository;
+
+import org.springframework.test.context.TestPropertySource;
+
+@TestPropertySource(locations = {
+        "classpath:/mongodbEmbeddedDisabled.properties",
+        "classpath:/secrets/mongodbConnectionString.properties"
+})
+public class HpanInitiativesAtomicOpsRepositoryImplTestIntegrated extends HpanInitiativesAtomicOpsRepositoryImplTest{
+
```

Comment:
if the test create some data, you have to delete here, otherwise you will let the db dirty